mars/deploy/oscar/tests/test_ray_dag.py

```diff
@@ -122,3 +126,22 @@ def test_sync_execute(config):
 @pytest.mark.asyncio
 async def test_session_progress(ray_start_regular_shared2, create_cluster):
     test_local.test_session_progress(create_cluster)
+
+
+@require_ray
+@pytest.mark.asyncio
+async def test_shuffle(ray_start_regular_shared2, create_cluster):
+    df = md.DataFrame(mt.random.rand(300, 4, chunk_size=100), columns=list("abcd"))
```

Comment:
Why not just run the `test_local.py::test_execute_describe`?

Reply:
we added `df.groupby` in this tests case, `test_execute_describe` doesn't contains `groupby`.

Reply:
Then, just extends the `test_execute_describe` case or add a new case? We are trying to get both of the backends to run the same test cases.

Reply:
Split it into `test_execute_describe` and `test_shuffle` test cases